tofu/geom/_comp.py

```diff
@@ -46,6 +46,18 @@
     'volume': 'abs',
 }
 
+
+def _check_float(var=None, varname=None, vardef=None):
+    if var is None:
+        var = vardef
+    if not type(var) in _LTYPES:
+        msg = (
+            "Arg {} must be a float!\n"
+            + "Provided: {}".format(type(var))
```

Comment:
what do you expect to have in the first `{ }`? maybe `var`? so something along the lines:
```
    "Arg {} must be a float!\n".format(var)
    + "Provided: {}".format(type(var))
```